standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionInfo.java

```diff
@@ -109,6 +109,13 @@ public void setProperty(String key, String value) {
     properties = propertiesMap.toString();
   }
 
+  public StringableMap getPropertiesMap() {
```

Comment:
not needed

Reply:
Please see [this](https://github.com/apache/hive/pull/3528#discussion_r952191010) comment.

Reply:
see above suggestion :)

Reply:
Done.